fix: code quality issues

withshubh opened the pull request:
#### Description
This PR fixes a few issues that were affecting the code quality.

#### Summary of changes
- Refactor unnecessary `else` / `elif` when `if` block has a `return` statement
- Remove unused imports
- Use literal syntax to create data structure

rmk135 commented on tests/unit/samples/wiringfastapi/web.py:
```diff
@@ -1,7 +1,6 @@
 import sys
 
 from fastapi import FastAPI, Depends
-from fastapi import Request  # See: https://github.com/ets-labs/python-dependency-injector/issues/398
```
Could you, please, leave this import? There was a bug when importing ``Request`` crashed wiring. Follow the link for the details.

withshubh replied:
reverted the change :sparkles:

rmk135 reviewed (changes requested):
@withshubh , thanks, that's fantastic! Two things before merging:

1) Take a look at my comment on removing ``fastapi.Request`` import.
2) Target the PR to develop branch.

withshubh commented:
@rmk135 Thanks for review! :sparkling_heart: 

I have made the changes!

withshubh commented:
I would like to share that I ran [DeepSource](https://deepsource.io/) analyzer on my fork of this repository to detect these issues. Have a look at the issues caught in this repository by DeepSource [here](https://deepsource.io/gh/withshubh/python-dependency-injector/issues/?category=recommended).

DeepSource is a code review automation tool that detects code quality issues and helps you to automatically fix some of them. You can use DeepSource to track test coverage, Detect problems in Dockerfiles, etc. in addition to detecting issues in code.

Can I submit an issue as a feature request for DeepSource integration or PR(consisting of a configuration file only)?
You can merge the configuration file to continuously analyze the repo and fix code quality issues.

rmk135 commented:
Thanks for the contribution @withshubh .

I've updated changelog https://github.com/ets-labs/python-dependency-injector/commit/95b0356edcea7c0f5f8a69377cd5166902ca6394 and added your name to the list of contributors https://github.com/ets-labs/python-dependency-injector/commit/3cf14c139f702bea0355a955d5ce888aa8f0aeae.

DeepSource tool looks good. The only concern I have now is that it seems too noisy. I'm good to give it a try in a recommendation manner. You're welcome to make a configuration file PR.

withshubh commented:
> Thanks for the contribution @withshubh .
> 
> I've updated changelog [95b0356](https://github.com/ets-labs/python-dependency-injector/commit/95b0356edcea7c0f5f8a69377cd5166902ca6394) and added your name to the list of contributors [3cf14c1](https://github.com/ets-labs/python-dependency-injector/commit/3cf14c139f702bea0355a955d5ce888aa8f0aeae).

Thank You so much :heart_eyes: 

> DeepSource tool looks good. The only concern I have now is that it seems too noisy. I'm good to give it a try in a recommendation manner. You're welcome to make a configuration file PR.

Here it is #407
